features/scenario_outlines.feature

```diff
@@ -93,3 +93,26 @@ Feature: Scenario Outlines and Examples
     When I run cucumber-js
     Then it fails
     And it runs 4 scenarios
+
+  Scenario: scenario outlines with tagged examples
+    Given a file named "features/scenario_outline.feature" with:
+      """
+      Feature: a feature
+        Scenario Outline: a scenario
+          Given a step <id>
+
+        @tag
```

Comment:
Would it be worth testing the multiple examples tables with different tags here?

Reply:
Yes, I'll do that.
We could even add two example tables with more tags to cover even more possibilities!?

Reply:
I think two scenarios - one with the simplest, one with the most complex permutation you mentioned - should be good.